### PR TITLE
fix: Dropdown menu results in internal scroll bar

### LINKED ISSUE #5307: Organizer Session Page: Dropdown menu results in internal scroll bar
When an organizer clicks on status pending, accepting etc. in the last row of a page suddenly an internal scroll bar shows up and the dropdown menu appears below other page elements.

Expected: No internal scroll bar. Elements should be on top of below elements.

Screenshot from 2020-10-14 08-02-37

### PR DESCRIPTION
**Pull request by nikolaytoplev**
Fixes #5307

#### Short description of what this resolves:
Table height is adjusted to wax and wane automatically and dropdown direction is calculated properly.

#### Changes proposed in this pull request:

-Table::Default component is patched to emit action of rendering end.
-Pre-calculation of dropdown direction is implemented
-CellButton component is modified to add direction option

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/tables/default.js [hunk 1]
```diff
@@ -4,4 +4,9 @@ import Component from '@ember/component';
 @classic
 export default class Tables extends Component {
 
+    didRender() {
+        if (this.rendered) {
+            this.rendered();
+        }
```
Comment (iamareebjamal) [inline app/components/tables/default.js:10]:
```suggestion
        this.rendered?.();
```

Comment (nikolaytoplev) [reply app/components/tables/default.js]: I am not very familiar with ?. operator yet.
Thank you.

**Review by mariobehling — APPROVED** [review]
Solution works for me. Thanks.

**Comment by iamareebjamal** [comment]
Unfortunately, this causes #5834, so have to revert

**Pull request merged**